lib/ex_admin/ex_admin.ex

```diff
@@ -180,6 +180,20 @@ defmodule ExAdmin do
     |> Keyword.get(resource_model)
   end
 
+
+  def get_registered_by_association(resource, assoc_name) do
+    resource_model = resource.__struct__
+    assoc_model = case resource_model.__schema__(:association, assoc_name) do
+      %{through: [link1, link2]} ->
+        resource |> Ecto.build_assoc(link1) |> Ecto.build_assoc(link2) |> Map.get(:__struct__)
+      %{queryable: assoc_model} ->
+        assoc_model
+      nil ->
```

Comment:
Should this match be `_` so it catches everything that does not match on the above clauses?